### PR TITLE
Deprecate authorization mode ABAC in v1.17

### PR DESCRIPTION
**Pull request by enj**
https://github.com/kubernetes/kubernetes/pull/82540

### CODE REVIEW

**Comment by enj** [comment]
@kubernetes/sig-auth-pr-reviews

**Comment by enj** [comment]
/hold

Until k/k PR is merged.

**Comment by sftim** [comment]
Hi @enj 

This should target the `dev-1.17` branch (once it exists), rather than `master`. For the documentation / website, a merge to `master` goes live as soon as the site is built.

File: content/en/docs/reference/access-authn-authz/abac.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ content_template: templates/concept
 weight: 80
 ---
 
+{{< note >}}
+{{< feature-state state="deprecated" for_k8s_version="1.17" >}}
```
Comment (sftim) [inline content/en/docs/reference/access-authn-authz/abac.md]: I recommend putting the feature-state shortcode _before_ the ` {{< note >}}` opening shortcode.

**Comment by sftim** [comment]
Some other places to deprecate ABAC:
- [/docs/reference/access-authn-authz/authorization/](https://github.com/kubernetes/website/blob/511e7ab5416eb67b5fbe1eb8427c8d788bb69954/content/en/docs/reference/access-authn-authz/authorization.md)
- [/docs/reference/access-authn-authz/controlling-access/](https://github.com/kubernetes/website/blob/511e7ab5416eb67b5fbe1eb8427c8d788bb69954/content/en/docs/reference/access-authn-authz/controlling-access.md)

File: content/en/docs/reference/access-authn-authz/abac.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/reference/access-authn-authz/abac.md]: Is it possible to either use the `feature-state` shortcode or add the deprecation comment in the note?

Comment (sftim) [reply content/en/docs/reference/access-authn-authz/abac.md]: Not sure what you're asking there @kbhawkey

Comment (kbhawkey) [reply content/en/docs/reference/access-authn-authz/abac.md]: I am questioning whether this page needs both the feature-state label _in addition to_ the note. The note provides more information.

Comment (kbhawkey) [reply content/en/docs/reference/access-authn-authz/abac.md]: Page preview:
https://deploy-preview-16318--kubernetes-io-master-staging.netlify.com/docs/reference/access-authn-authz/abac/

Right, I _think_ the note is clear.

**Comment by sftim** [comment]
v1.17 is released
It's OK to open a similar PR if ABAC becomes deprecated for, eg, v1.18
/close

**Pull request closed without merging**